keyform/forms.py

```diff
@@ -66,6 +66,11 @@ class Meta:
             'alert_statuses': CheckboxSelectMultiple,
         }
 
+    def clean_email(self):
+        email = self.cleaned_data.get('email')
+        return email.lower()

```

Comment:
This will only work if the emails in the database are already lower case. Which they aren't. We should either have code that will fix this or create an issue so that we make sure to remember to fix it ourselves.

Reply:
#66 OK.